CCallback.h

```diff
@@ -80,6 +80,15 @@ class IGameActionCallback
 	virtual void buildBoat(const IShipyard *obj) = 0;
 };
 
+class IGameBulkActionCallback // To implement high-level army management actions

```

Comment:
What is the reason having it as a separate interface? Do you plan to implement it alone? I feel like it will always be implemented together with IGameActionCallback.

Reply:
Ok